### DISCUSSION TITLE
Missing Data for the Step Change in Experiment

### DISCUSSION THREAD

**Discussion by JayYDG** [pybamm-team/PyBaMM#3837] [Q&A]
Experiment:
```
['Rest for 0.5 hour (1 minute period)',
 'Charge at 1C until 4.2 V (1 minute period)',
 'Hold at 4.2V until C/10 (1 minute period)',
 'Rest for 0.5 hour (1 minute period)',
 'Discharge at 1C until 2.5 V (1 minute period)',
 'Rest for 0.5 hour (1 minute period)']
```

Current plot
![image](https://github.com/pybamm-team/PyBaMM/assets/110075687/db1a20d3-3af9-4258-9b72-cc281a42dd65)

I cycle the DFN model with this experiment and stored the result. Just wonder why the sol2 didn't include the 1st point the CC charging but the sol1 did. Is there any solution to enforce the solution include the first point after the step change?

**Comment by valentinsulzer** [comment]
How are sol1 and sol2 different, can you share the full code (or ideally a minimum reproducible example)?
